animations/lib/src/basics/05_animated_builder.dart

```diff
@@ -13,7 +13,7 @@ class AnimatedBuilderDemo extends StatefulWidget {
 
 class _AnimatedBuilderDemoState extends State<AnimatedBuilderDemo>
     with SingleTickerProviderStateMixin {
-  static const Color beginColor = Colors.deepPurple;
+  static const Color beginColor = Colors.indigoAccent;
```

Comment:
Any particular rationale for changing the color here?

Reply:
No @RedBrogdon . It was just same colour as a AppBar color so I thought to apply new color!
If it's not good then We can undo the changes!
What do you think?

Reply:
It's not a critical thing -- I just wanted to know the rationale. If anything, indigo is more of an opposite to orange, so this change seems fine to me:

https://www.google.com/search?q=color+wheel+indigo